conf/wes.config

```diff
@@ -0,0 +1,12 @@
+/*
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+    Nextflow config file for running the pipeline on the CMGG infrastructure
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+*/
+
+params {
+    exomedepth = true
+    annotate = true
+    roi_auto = "${params.genomes_base}/Hsapiens/GRCh38/regions/Homo_sapiens.GRCh38.105.chr_protein_coding_basic_sorted_merged_autosomal.bed"
+    roi_chrx = "${params.genomes_base}/Hsapiens/GRCh38/regions/Homo_sapiens.GRCh38.105.chrX_protein_coding_basic_sorted_merged.bed"
```

Comment:
also add vep_cache here?

Reply:
It's in the `main.nf` and will be fetched from the genomes config :)